### ISSUE TITLE
Dividing lines occur when a father's 'Container' does not match the color of his children's 'color'

### ISSUE THREAD

**Issue by beilo** [flutter/flutter#39815]
Dividing lines occur when a father's 'Container' does not match the color of his children's 'color'.
Replacing 'ListView' with 'Column mainAxisSize: mainaxission.min' also fails to solve this problem
```
  Container _screenListBuild() {
    return Container(
      height: double.infinity,
      width: double.infinity,
      color: Color(0x7F333333),
      child: ListView(
        itemExtent:null,
        shrinkWrap:false,
        children: <Widget>[
          Container(
            height: 45, // Dividing lines
            child: Text("111"),
            color: Colors.white,
            padding: EdgeInsets.only(left: suW(20), right: suW(20)),
          ),
          Container(
            height: 44, // not Dividing lines
            child: Text("111"),
            color: Colors.white,
            padding: EdgeInsets.only(left: suW(20), right: suW(20)),
          ),Container(
            height: 45, // Dividing lines
            child: Text("111"),
            color: Colors.white,
            padding: EdgeInsets.only(left: suW(20), right: suW(20)),
          ),Container(
            height: 44, // not Dividing lines
            child: Text("111"),
            color: Colors.white,
            padding: EdgeInsets.only(left: suW(20), right: suW(20)),
          ),
        ],
      ),
    );
  }
```
![img](https://upload-images.jianshu.io/upload_images/1794988-5ad6ef4b9977579d.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
![mobile](https://upload-images.jianshu.io/upload_images/1794988-3556dfab084e2a87.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)

```
Last login: Wed Sep  4 16:48:54 on ttys002
➜  ~ flutter doctor -v
[✓] Flutter (Channel beta, v1.8.3, on Mac OS X 10.14.4 18E226, locale
    zh-Hans-CN)
    • Flutter version 1.8.3 at /Users/beilo/Documents/develop/flutter
    • Framework revision e4ebcdf6f4 (6 周前), 2019-07-27 11:48:24 -0700
    • Engine revision 38ac5f30a7
    • Dart version 2.5.0 (build 2.5.0-dev.1.0 0ca1582afd)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/beilo/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.2, Build version 10E125
    • CocoaPods version 1.7.3

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 38.2.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[!] IntelliJ IDEA Community Edition (version 2019.1.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
```

thank you

**Comment by HansMuller** [comment]
This looks like a known problem with antialiasing and mapping from logical pixels to device pixels. There's an open issue for a documentation fix: #14288

More info about the problem can be found here:
#14288 (comment)
#17084 (comment)
#15035 (comment)

**Comment by goderbauer** [comment]
Closing as a dupe of https://github.com/flutter/flutter/issues/14288#issuecomment-361375791, which has more information and a work-around.
